### ISSUE TITLE
AWS Autoscaling: How to specify min/max nodes with autodiscovery?

### ISSUE THREAD

**Issue by trondhindenes** [kubernetes/autoscaler#1013]
From what I can see in the docs, I can either specify a "node group" (which takes min/max nodes) or use autodiscovery tags. It's not clear from the documentation how to specify min/max nodes when autodiscovery is used. It would be awesome if that was clarified.

**Comment by aleksandra-malinowska** [comment]
See usage described in #462, which added support for it on GCE.

**Comment by trondhindenes** [comment]
Does that mean AWS support was already in place, or that there isn't support for it on AWS? It's not clear to me from looking at #462.

**Comment by trondhindenes** [comment]
I tried the same parameter structure (wow did I just write that) as in the PR, and that's not working at all:
`--node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled,kubernetes.io/cluster/rtv-dev-cluster02,minNodes=1,maxNodes=10`
It works fine if I removed the `minNodes`/`maxNodes` thingy.

**Comment by johanneswuerbach** [comment]
If you are using autodiscovery the min/max values are taken from the discovered ASGs and can be adjusted on the fly by changing the values of the AWS ASG itself.

You can override this by manually specifying an ASG and providing min/max as autoscaler parameters. This always takes precedence over autodiscovered values.

**Comment by trondhindenes** [comment]
Thanks @johanneswuerbach . So what you're saying is that as long as `auto-discovery` is enabled, cluster-autoscaler will _always_ respect the limits set in the ASG.

**Comment by johanneswuerbach** [comment]
Correct, the autoscaler is only modfiying the desired size value of an asg, never min or max.

**Comment by mf-lit** [comment]
This confused me too, PR opened.
